jsonnet/jsonnetfile.lock.json

```diff
@@ -8,7 +8,7 @@
                     "subdir": "contrib/kube-prometheus/jsonnet/kube-prometheus"
                 }
             },
-            "version": "a24386e2d5e66de353cb0e939936041fa3977283"
+            "version": "5318247b242823e6f45266f678855f0eef215f05"
```

Comment:
The lock file now contains all versions of all resolved dependencies. As a hack you can specify this version in the `jsonnetfile.json`, then remove `jsonnetfile.lock.json` and the `vendor/` directory inside `jsonnet/`, and then run `jb install`.